modules/logging-release-notes-5-8-2.adoc

```diff
@@ -0,0 +1,28 @@
+//module included in logging-5-8-release-notes.adoc
+:content-type: REFERENCE
+[id="logging-release-notes-5-8-2"]
+= Logging 5.8.2
+This release includes link:https://access.redhat.com/errata/RHSA-2024:0271[OpenShift Logging Bug Fix Release 5.8.2].
+
+[id="logging-release-notes-5-8-2-known-issues"]
+== Known issue
+Currently, the log alert is not available on an IPv6-enabled cluster. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/LOG-4890[LOG-4890])
```

Comment:
As mentioned elsewhere the updated text is on: https://issues.redhat.com/browse/LOG-4890